frontend/tests/search.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { test, expect } from "@playwright/test";
+import { navigateTo } from "./common";
+
+
+test("Search", async ({ page }) => {
+    await navigateTo("/", page);
+    await page.waitForSelector("nav");
+    const searchField = page.locator("_react=SearchField").locator("input");
+
+    await test.step("Should be focusable by keyboard shortcut", async () => {
+        await page.keyboard.press("s");
+        await expect(searchField).toBeFocused();
```

Comment:
Nothing wrong with this, just reminded me: we certainly need tests of typing something in a different input that contains `s` and making sure in that case, it does NOT jump to the search bar.